### PR TITLE
feat(current-bound): add isCurrentBound for check dependency only in …

### PR DESCRIPTION
**Pull request by dmitry-nizovsky**
add isCurrentBound for check dependency only in current container

## Description

add api method for user can check already registered dependency in container without bubbling checking to parent

## Motivation and Context

Then I did integration InversifyJS with ReactJs I found one issue.
I cannot check bound dependency only current container.

## Types of changes

- [x] Updated docs / Refactor code / Added a tests case (non-breaking change)
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] I have updated the changelog.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "inversify",
-  "version": "5.1.0",
+  "version": "5.1.1",
```
Comment (notaphplover) [inline package-lock.json]: Version should not be updated

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "inversify",
-  "version": "5.1.1",
+  "version": "5.1.2",
```
Comment (notaphplover) [inline package.json]: Same

File: CHANGELOG.md [hunk 3]
```diff
@@ -21,6 +21,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 - only inject decorator can be applied to setters #1342
 - Container.resolve should resolve in that container #1338
+
+## [5.1.2] - 2021-06-15
```
Comment (notaphplover) [inline CHANGELOG.md]:
```suggestion
## [Unreleased]
```

File: src/container/container.ts [hunk 4]
```diff
@@ -249,6 +249,11 @@ class Container implements interfaces.Container {
         return bound;
     }
 
+    // check binding dependency only in current container
+    public isCurrentBound(serviceIdentifier: interfaces.ServiceIdentifier<any>): boolean {
```
Comment (notaphplover) [inline src/container/container.ts]:
```suggestion
    public isCurrentBound<T>(serviceIdentifier: interfaces.ServiceIdentifier<T>): boolean {
```

What about this? We try not to use any these days

**Review by notaphplover — CHANGES_REQUESTED** [review]
Looks good, I just suggested some minor changes

**Comment by dmitry-nizovsky** [comment]
Hello @notaphplover,
I did fix, Could you check again please?

**Review by notaphplover — APPROVED** [review]
LGTM!

**Comment by dmitry-nizovsky** [comment]
Hello @notaphplover,
What should I do to merge this PR?

**Comment by dmitry-nizovsky** [comment]
Hi @notaphplover,
Could you help me with merge PR please?

**Pull request merged**